event/event.go

```diff
@@ -5,35 +5,39 @@ package event
 
 import (
 	"context"
+	"fmt"
 	"sync"
 
 	golog "github.com/ipfs/go-log"
 )
 
-var log = golog.Logger("event")
+var (
+	log = golog.Logger("event")
+
+	// ErrBusClosed indicates the event bus is no longer coordinating events
+	// because it's parent context has closed
+	ErrBusClosed = fmt.Errorf("event bus is closed")
```

Comment:
Is this something we should handle on every subscription?

Reply:
If we see this error, it's a bug. The event bus should be wired to the same long-running context that all subsystems are wired to, and all should close at the same time. In that scenario this error will appear if a subsystem doesn't obey context closure & continues to publish events.

So no, I don't think it should need to be explicitly handled.